docs/UserGuide.md

```diff
@@ -72,13 +72,13 @@ about animals in a shelter easily.
 #### Adding an animal: `add`
 Adds an animal to the address book.
 
-Format: `add n/NAME i/ID db/DOB s/SEX b/BREED da/DOA`
+Format: `add n/NAME i/ID db/DOB b/BREED da/DOA g/SEX s/SPECIES `
 
 * ID must be a unique 4 digit number.
 
 Examples:
-* `add n/Pookie i/1234 db/28-2-2022 s/male b/Golden Retriver da/23-5-2022`
-* `add n/Tofu i/1242 db/21/4/2023 s/female b/British Shorthair da/25-5-2022`
+* `add n/Pookie i/1234 db/28-2-2022 g/male b/Golden Retriver da/23-5-2022 s/dog`
```

Comment:
Not necessary but would it be better if the example follows the given format above? It could make users less confused but just a suggestion!

Reply:
For this, I think it should be fine given one of our features:
Parameters can be in any order.<br>
  e.g. if the command specifies `n/NAME db/DOB`, `db/DOB n/NAME` is also acceptable.